frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -312,6 +319,52 @@ const HostSummary = ({
     );
   };
 
+  const renderOperatingSystemSummary = () => {
+    // No tooltip if minimum version is not set, including all Windows, Linux, ChromeOS operating systems
+    if (!osUpdatesData?.minimum_version) {
```

Comment:
Would it be cleaner to have this be a ternary on line 345 when defining `value`?

Reply:
I mean line 354

Reply:
looks much cleaner! Ty!